packages/handlersjs-http/lib/errors/bad-request-http-error.ts

```diff
@@ -42,6 +42,8 @@ export class BadRequestHttpError extends HttpError {
 
   static isInstance(error: unknown): error is BadRequestHttpError {
 
+    this.logger.info(`Checking if ${error} is an instance of ${this.name}: `, HttpError.isInstance(error) && error.statusCode === 400);
```

Comment:
Would put the `isInstance` in a const so it doesn't have to be calculated twice.
(applies to other files as well)